pkg/api/testing/serialization_test.go

```diff
@@ -72,7 +72,7 @@ func Convert_v1beta1_ReplicaSet_to_api_ReplicationController(in *v1beta1.Replica
 	}
 
 	intermediate2 := &v1.ReplicationController{}
-	if err := k8s_api_v1.Convert_extensions_ReplicaSet_to_v1_ReplicationController(intermediate1, intermediate2, s); err != nil {
+	if err := k8s_api_v1.Convert_extensions_ReplicaSet_To_v1_ReplicationController(intermediate1, intermediate2, s); err != nil {
```

Comment:
is it worth it, `to`->`To`?

Reply:
ic, only a couple which were misnamed.

Reply:
Right, which means they were "accidentally" (not used) omitted from static analysis.  I strengthened the static check so that you can't get into this scenario anymore without calling it something other than `Convert_`.